src/main/scala/com/cloudera/sparkts/TimeSeries.scala

```diff
@@ -287,6 +297,21 @@ object TimeSeries {
     }
     new TimeSeries[K](index, mat, keys)
   }
+
+  def timeSeriesFromVectors[K](vectors: Seq[Vector[Double]],
```

Comment:
Bring `vectors` down to the next line and indent all the arg lines four spaces ahead of `def`.

Also, would it make sense for this to accept an `Iterable`?